### PR TITLE
Update to 1.77

### PR DESCRIPTION
**Pull request by Veykril**

### CODE REVIEW

File: src/lexical-elements.rst [hunk 1]
```diff
@@ -703,6 +704,112 @@ The :t:`type` of a :t:`raw byte string literal` of size ``N`` is ``&'static
...
+
+.. rubric:: Legality Rules
+
+:dp:`fls_nPI7j0siGP8G`
+A :t:`simple c string literal` is a :t:`c string literal` where the characters are
+:t:`Unicode` characters.
+
+:dp:`fls_Ae7LM4Wg0NA7`
+The :t:`type` of a :t:`simple string literal` is :std:`&'static
+[core::ffi::CStr]`.
```
Comment (pietroalbini) [inline src/lexical-elements.rst:760]: Shouldn't this be `&'static core::ffi::CStr`?

File: src/lexical-elements.rst [hunk 2]
```diff
@@ -703,6 +704,112 @@ The :t:`type` of a :t:`raw byte string literal` of size ``N`` is ``&'static
...
+
+.. rubric:: Legality Rules
+
+:dp:`fls_gLrei65i8Uzq`
+A :t:`raw c string literal` is a :t:`simple c string literal` that does not
+recognize :t:`[escaped character]s`.
+
+:dp:`fls_9nJHsg9dCi66`
+The :t:`type` of a :t:`simple string literal` is :std:`&'static
+[core::ffi::CStr]`.
```
Comment (pietroalbini) [inline src/lexical-elements.rst:803]: Shouldn't this be `&'static core::fii::CStr`?

**Review by pietroalbini — CHANGES_REQUESTED** [review]

File: src/lexical-elements.rst (hunk 1 above)
Comment (Veykril) [reply src/lexical-elements.rst:760]: The `[]` is the processing syntax for the std linkage, it does render as `&'static core::ffi::CStr`

**Comment by Veykril** [comment]
Hmm, for some reason the `CStringLiteral` syntax rule is not being linked to

**Comment by Veykril** [comment]
Seems to be a bug in our tooling, will investigate that later, should not block this PR.

File: src/lexical-elements.rst (hunk 1 above)
Comment (pietroalbini) [reply src/lexical-elements.rst:760]: ...right :sweat_smile:

**Review by pietroalbini — APPROVED** [review]
bors merge

**Pull request merged**